### ISSUE TITLE
Prevent changing indentation in composer.json

### ISSUE THREAD

**Issue by dahaupt** [rectorphp/rector#5224]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.9.9              |
| Installed as   | composer dependency ([sabbelasichon/typo3-rector](https://github.com/sabbelasichon/typo3-rector))                         |

Since #5074 (I guess), the `composer.json` is included in the refactoring process. In our case, this breaks with our configuration within `.editorconfig` as rector now wants to change the indentation. We like to use tabs instead of spaces. As we run rector as part of our CI pipeline, this is of course inconvenient if we want to keep our current indentation style.

## Expected Behaviour
Possibility to enable or disable a rule which is responsible for this change. Alternatively make indentation configurable. 

Also, I tried to completely skip the file `composer.json`, which did not work:
```
$parameters->set(Option::SKIP, [
        __DIR__ . '/composer.json',
    ]);
```

**Comment by TomasVotruba** [comment]
cc @lulco

**Comment by lulco** [comment]
If no composer rectors are enabled there should be no change at all:
- https://github.com/rectorphp/rector/pull/5198/files#diff-6ac00ca5ab4c40e2d2ed6f061d33c2af8d3304ca791183fadc6377c63a014873R76

- https://github.com/rectorphp/rector/pull/5198/files#diff-419022eebd8ba2552dcfdf1d9151489fc372d04c88b3ab093bea7c64962938feR32

**Comment by lulco** [comment]
And I think indentation is global problem - also in PHP, yaml, neon files. Isn't it?

**Comment by dahaupt** [comment]
> If no composer rectors are enabled there should be no change at all:
> 
> * https://github.com/rectorphp/rector/pull/5198/files#diff-6ac00ca5ab4c40e2d2ed6f061d33c2af8d3304ca791183fadc6377c63a014873R76

@lulco I've reported this issue using v0.9.9 and those changes are available since v0.9.10, I will try to update the version ASAP to test the behavior again. 

About the global problem of indentation: In our case rector changes **only** the indentation without further modifications or changes / fixes. It should not report indentation only issues IMO.

**Comment by dahaupt** [comment]
I can confirm that the issue is solved with v0.9.10. Thanks a lot for your fast feedback and please excuse the confusion.
